roles/change-paths/defaults/main.yml

```diff
@@ -3,4 +3,10 @@ all_sar_commands: [ 'sar', 'sarl', 'sarm', 'sarv', 'sarjs' ]
 sarl_commands: [ 'sarl', 'sarm' ]
 sar_fulltext_commands: [ 'sar', 'sarv', 'sarjs' ]
 protocols: [ 'https://', 'http://' ]
-link_attributes: [ 'url="', 'href="', 'src="' ]
+link_attributes:
+  - url="
```

Comment:
Actually, I'm not sure of the use case of `url=` here. We have:
- `<img src="...`
- `<a href="...`

But what is the `url` attribute? Did we add this when we thought that `sar` was doing find-and-replace on CSS files?

There's also this: https://stackoverflow.com/questions/2725156/complete-list-of-html-tag-attributes-which-have-a-url-value, but I think before we get to that point that we say, "That's it. This is how fine-grained our sifter is; anything else is an edge case."